examples/kv_addr_book/include/kv_addr_book.hpp

```diff
@@ -70,14 +68,14 @@ class [[eosio::contract]] kv_addr_book : public eosio::contract {
 
    struct [[eosio::table]] address_table : eosio::kv::table<person, "kvaddrbook"_n> {
       // unique indexes definitions
-      // 1. they are defined for just one property of the kv_table parameter type (person)
-      // 2. unique indexes for multiple properties of the kv_table parameter type
+      // 1. they are defined for just one property of the kv::table parameter type (person)
+      // 2. unique indexes for multiple properties of the kv::table parameter type
       //    are defined with the help of a pair or a tuple; a pair if the index has 
       //    two properties or a tuple in case of more than two
-      index<name> account_name_uidx {
+      index<eosio::name> account_name_uidx {
```

Comment:
Is there any reason not using KV_NAMED_INDEX?

Reply:
yes, we want to show/exemplify all possible ways to do it.